ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityEmailInput.component.ts

```diff
@@ -160,7 +160,7 @@ export class ActivityEmailInput implements OnInit, OnChanges, OnDestroy {
                 return null;
             }
             // Set error on matchingControl if validation fails
-            if (control.value !== matchingControl.value) {
+            if (control.value.toLowerCase !== matchingControl.value.toLowerCase) {
```

Comment:
Even though email address is not a case sensitive in general, it seems (to me) that it is more clear for a user to have the same values for `email` and `confirmEmail` fields. 
I have not found any requirements to change the behavior in related tickets DDP-7520 / DDP-7501.
Was a demand to do it ?